decentraland/unity-renderer
# Feat: added new placeholder passport panel behind Feature Flag

## What does this PR change?

Fix: #3312
Adds a new panel placeholder for the new passport, it is behind a feature flag that toggles the old and new passport flow.

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/new-passport-panel
2. Verify that the old passport appears when clicking on selected users
3. Go to: https://play.decentraland.zone/?renderer-branch=feat/new-passport-panel&ENABLE_new_avatar_flow=
4. Verify that a new empty panel appears when clicking on selected users

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
